### PR TITLE
HA_REPO_ENABLED variable

### PR DESCRIPTION
**Pull request by alphacc**
Enable -ha centos repository for each SIG if needed.
Default to true for cloud-openstack-*

Please review

### CODE REVIEW

File: templates/cbs-tools/scripts/create_sig.sh [hunk 1]
```diff
@@ -221,6 +221,10 @@ do
                     fi
                     if [[ "x$DIST" == "x8" ||  "x$DIST" == "x8s" ]]
                     then
+                        if ( $HA_REPO_ENABLED )
```
Comment (arrfab) [inline templates/cbs-tools/scripts/create_sig.sh:224]: HA_REPO_ENABLED has no default ? I see you declare it only in the openstack config, but should then be declared as "false" in default config ? 
Probably in this line for common : https://github.com/CentOS/ansible-role-kojihub/blob/master/templates/cbs-tools/scripts/create_sig.sh#L262 ?

**Review by arrfab — CHANGES_REQUESTED** [review]
Check the default 'false' value for HA_REPO_ENABLED ?

**Comment by alphacc** [comment]
Good point, i added it at the default per project setting.

**Comment by arrfab** [comment]
Good, let's merge :)

**Pull request merged**